pkg/agent/flags/flag_values_stored.go

```diff
@@ -0,0 +1,47 @@
+package flags
+
+import (
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/agent/types"
+)
+
+type storedFlagValues struct {
```

Comment:
I had to go back and check the docs a couple times to figure out/understand where the stored flag values came from/what they were for. I think the naming was a little confusing for me because the fact that they live in a KVStore isn't the distinguishing thing about them -- it's that they're set by k2/the control server, i.e. a cloud override to the flags shipped with the installer. Could we consider a name that explains their purpose/source? Maybe `cloudFlagValues` or `cloudControlledFlagValues`? (Not suggesting we use `cloud override` because `override` already has another meaning here 😅 )

Reply:
I got rid of stored flag value stuff, and renamed this area to `getControlServerValue`.